missioncontrol/model/models.py

```diff
@@ -0,0 +1,80 @@
+from django.db import models
+
+
+class Platform(models.Model):
+    '''
+    Represents a platform e.g. "windows"
+    '''
+    name = models.CharField(max_length=100, unique=True)
+    telemetry_name = models.CharField(max_length=100, unique=True)
+
+    class Meta:
+        db_table = 'platform'
+
+
+class Channel(models.Model):
+    '''
+    Represents a release channel e.g. "beta", "esr", "nightly"
+    '''
+    name = models.CharField(max_length=100, unique=True)
+    update_interval = models.DurationField()
+
+    class Meta:
+        db_table = 'channel'
+
+
+class Build(models.Model):
+    '''
+    Represents a specific build of the product
+    '''
+    platform = models.ForeignKey(Platform)
+    channel = models.ForeignKey(Channel)
+    build_id = models.CharField(max_length=14)
+    version = models.CharField(max_length=10)
+
+    class Meta:
+        db_table = 'build'
+        unique_together = ('platform', 'channel', 'build_id', 'version')
+
+
+class Measure(models.Model):
+    '''
+    Represents a type of measure (e.g. "main_crashes") on a particular channel
+
+    Some measures occur only on specific types of platform (e.g. "gpu_crashes"
+    is Windows-only)
+    '''
+    name = models.SlugField(max_length=100)
```

Comment:
Will all the measures validate correctly against the `SlugField` validation rules?

Reply:
Yes, the slug field validation rules seem like a reasonable sanity check on what we're doing here.